python/py-smmap/Portfile

```diff
@@ -1,24 +1,22 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
please don't remove the modeline and make incorrect changes to the whitespace (should be 4 spaces)

Reply:
thanks for re-adding the `modeline`. However, you still make unnecessary (and incorrect) white space changes in this update; please revert.